pylintrc

```diff
@@ -310,7 +310,7 @@ mixin-class-rgx=.*Mix[Ii]n
 # List of members which are set dynamically and missed by pylint inference
 # system, and so shouldn't trigger E0201 when accessed. Python regular
 # expressions are accepted.
-generated-members=REQUEST,acl_users,aq_parent,argparse.Namespace
+generated-members=REQUEST,acl_users,aq_parent,argparse.Namespace,ast\.([mM]atch.*|pattern)
```

Comment:
? Shouldn't this be in a different PR/commit?

Reply:
Need this to fix a `no-member` issue now that the string annotations are replaced with `from __future__ import annotations`, see https://github.com/PyCQA/pylint/issues/6594.

Reply:
Ah! That explains it!